### PR TITLE
docs: init migration notes - FRONT-1835

### PR DESCRIPTION
**Pull request by kalinchernev**
Starting point with an attempt to be as concise as possible.

### CODE REVIEW

File: docs/Migrating-v3.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+The following guidelines aim to facilitate migration between ECL v2 to v3.
+
+## Deprecated components haven't been migrated
+
+The following have been kept in v2 in order to avoid breaking changes, but will no longer be available in v3:
+
+Components:
+
+- Accordion (deprecated since ECL 2.6.0)
```
Comment (emeryro) [inline docs/Migrating-v3.md]: we may specify that this is the old accordion version (we still have an accordion in ECL 3, which correspond to component accordion2 of ECL 2)

File: docs/Migrating-v3.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+Example: `@ecl/ec-component-accordion` => `@ecl/vanilla-component-accordion`
+
+- Twig templates packages have the following change in pattern: `@ecl-twig/(ec|eu)-component-{component_name}` has become `@ecl/twig-component-{component_name}`.
+
+Example: `@ecl-twig/ec-component-accordion` => `@ecl/twig-component-accordion`
+
+- Resources' packages have the following change in pattern: `@ecl/(ec|eu)-resources-{resource_name}` has become `@ecl/resources-(ec|eu)-{resource_name}`
+
+Example: `@ecl/ec-resources-logo` => `@ecl/resources-ec-logo`
+
```
Comment (emeryro) [inline docs/Migrating-v3.md:38]: should we also mention the renaming of specs packages?
`@ecl/ec-specs-accordion` => `@ecl/specs-component-accordion`

File: docs/Migrating-v3.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+| `ecl-u-fs-*`                                                                                                              | use `@ecl/vanilla-utility-typography` and `ecl-u-type-*` discouraged                                                         |
+| `ecl-u-ratio-*`                                                                                                           | use `@ecl/ec-utility-media` and `ecl-u-media-ratio-*`                                                                        |
+| `ecl-u-text-*`                                                                                                            | use `@ecl/vanilla-utility-typography` and `ecl-u-type-*`                                                                     |
+
+Please note that not all changes listed above will be breaking changes for your project as it depends on how CSS classes have been applied through templates.
+
+If you have applied any of these CSS classes manually or programatically in your website, they please pay extra attention.
```
Comment (emeryro) [inline docs/Migrating-v3.md]:
```suggestion
If you have applied any of these CSS classes manually or programatically in your website, then please pay extra attention.
```

**Review by emeryro — COMMENTED** [review]
Thanks for this.
We should keep this PR open for some time and update the documentation until we reach ECL 3.0

File: docs/Migrating-v3.md (hunk 2 above)
Comment (kalinchernev) [reply docs/Migrating-v3.md:38]: I have deliberately left specs out of this document. Specs are used only internally and adding information about them will only add noise. I doubt we'll even publish them on npm now having twig templates in the repo.

**Comment by kalinchernev** [comment]
I agree this is only a starting point of the document and it will be elaborated in time. I suggest we merge it in the main v3 branch when approved and it will be extended in time similarly to any other piece of code in the repo

**Review by emeryro — APPROVED** [review]

**Pull request merged**